index.html

```diff
@@ -90,10 +90,22 @@ <h3>
       <dfn>RTCAudioReceiverStats</dfn> dictionary non-standardized members
     </h3>
     <pre class="idl">partial dictionary RTCAudioReceiverStats {
+      unsigned long long delayedPacketOutageSamples;
       unsigned long long jitterBufferFlushes;
 };</pre>
     <dl data-link-for="RTCAudioReceiverStats" data-dfn-for="RTCAudioReceiverStats" class=
     "dictionary-members">
+      <dt>
+        <dfn><code>delayedPacketOutageSamples</code></dfn> of type
+        <span class="idlMemberType"><a href="https://heycam.github.io/webidl/#idl-unsigned-long-long">unsigned long long</a></span>
+      </dt>
+      <dd>
+        <p>
+          Experimental stat which is available <a href="https://docs.google.com/document/d/1stYIZhEmDZ7NJF9gjjsM66eLFJUdc-14a3QutrFbIwI/edit#" title="experiment description">under origin trial in Chromium starting from M72 version</a>.
+          Measures the number of audio samples that were concealed due to jitter buffer underrun caused by delayed packets.
```

Comment:
What about jitter buffer underrun caused by clock drift? Counted or not counted?

This stat definition seems to carry strong assumptions about the cause of the concealed samples; I'm not sure we know the cause when applying concealment.

Reply:
We don't know the cause of the concealment when applying it, but we know the cause of it afterwards when we receive the next packet without a jump in sequence numbers.

Reply:
@henbos